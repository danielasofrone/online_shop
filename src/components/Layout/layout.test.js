// import React from 'react';
// import {shallow} from 'enzyme';
// import toJson from 'enzyme-to-json';
// import Layout from './Layout';

// describe('Layout', () => {
//   it('renders without crashing given the required props', () => {
//     const wrapper = shallow(
//       <Layout>
//         <>Mock Container</>
//       </Layout>
//     );
//     expect(toJson(wrapper)).toMatchSnapshot();
//   });
// });
