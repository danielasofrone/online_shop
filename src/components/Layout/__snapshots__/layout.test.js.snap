// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders without crashing given the required props 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      <img
        alt="PiNCAMP Logo"
        src="logo.svg"
        title="PiNCAMP"
      />
    </styled.div>
    <styled.div>
      Mock Container
    </styled.div>
  </styled.div>
</styled.div>
`;
